cmf-cli/Enums/PackageType.cs

```diff
@@ -68,6 +68,10 @@ public enum PackageType
         /// <summary>
         /// The database
```

Comment:
```suggestion
        /// The IoT Data
```

Reply:
Fixed
Also change the comment of the None item